### PR TITLE
[DEV-13004] Feature - Text highlights & callout blocks

### PR DESCRIPTION
**Pull request by e1himself**
https://github.com/prezly/slate/assets/370680/b9ee230d-49ce-40c1-a213-b1a41355a191

### CODE REVIEW

File: packages/slate-types/src/lib/isAlignableElement.ts [hunk 1]
```diff
@@ -1,8 +1,15 @@
...
 export type AlignableNode = HeadingNode | ListNode | QuoteNode | ParagraphNode;
 
 export function isAlignableElement(node: any): node is AlignableNode {
-    return isParagraphNode(node) || isHeadingNode(node) || isQuoteNode(node) || isListNode(node);
+    return (
+        isParagraphNode(node) ||
+        isHeadingNode(node) ||
+        isQuoteNode(node) ||
+        isListNode(node) ||
+        CalloutNode.isCalloutNode(node)
```
Comment (MohammadxAli) [inline packages/slate-types/src/lib/isAlignableElement.ts:13]: _Comment in an unrelated place_

Btw, I believe we should also add support for this new node in Automatic Story Translation lambdas as well.

https://github.com/prezly/prezly-auto-translation-lambdas/blob/46e8274c0164ed33fec5baacc3e51119a8bf92c0/src/lib/serialize/serializeToXml.ts#L28-L32

File: packages/slate-editor/src/extensions/callout/components/CalloutElement.tsx [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+                        skinTonesDisabled={true}
+                        suggestedEmojisMode={SuggestionMode.RECENT}
+                        emojiStyle={EmojiStyle.APPLE}
+                        width={275}
+                        height={350}
+                    />
+                    <label className={styles.NoIconOption}>
+                        <input
+                            type="checkbox"
+                            style={{}}
```
Comment (MohammadxAli) [inline packages/slate-editor/src/extensions/callout/components/CalloutElement.tsx:178]:
```suggestion
```

**Review by MohammadxAli — APPROVED** [review]
👍

File: packages/slate-types/src/lib/isAlignableElement.ts (hunk 1 above)
Comment (e1himself) [reply packages/slate-types/src/lib/isAlignableElement.ts:13]: Good remark. Thanks! :+1:

File: packages/slate-editor/src/extensions/callout/components/CalloutElement.tsx (hunk 2 above)
Comment (e1himself) [reply packages/slate-editor/src/extensions/callout/components/CalloutElement.tsx]: 392182e2fb96c5e43e5292c2a509ae0a477064c2

File: packages/slate-types/src/lib/isAlignableElement.ts (hunk 1 above)
Comment (e1himself) [reply packages/slate-types/src/lib/isAlignableElement.ts:13]: DEV-13045

Comment (e1himself) [reply packages/slate-types/src/lib/isAlignableElement.ts:13]: https://github.com/prezly/prezly-auto-translation-lambdas/pull/10

**Pull request merged**